kuma/payments/urls.py

```diff
@@ -5,23 +5,11 @@
 from . import views
 
 lang_urlpatterns = [
-    url(r'^success/?$',
-        views.confirmation, {'status': 'succeeded'},
-        name='payment_succeeded'),
-    url(r'^error/?$',
-        views.confirmation, {'status': 'error'},
-        name='payment_error'),
-    url(r'^recurring/success/?$',
-        views.confirmation, {'status': 'succeeded', 'recurring': True},
-        name='recurring_payment_succeeded'),
-    url(r'^recurring/error/?$',
-        views.confirmation, {'status': 'error', 'recurring': True},
-        name='recurring_payment_error'),
     url(r'^recurring/?$',
-        views.contribute_recurring_payment_initial,
+        views.contribute,
         name='recurring_payment_initial'),
     url(r'^recurring/subscription/?$',
-        views.contribute_recurring_payment_subscription,
+        views.contribute,
```

Comment:
It seems fine by me to do it this way without a redirect. If we ever turned contributions back on, these URLs would become live again. And if we discontinue the program for good, then view.contribute could redirect to a new "payments/discontinued" page.